spacelift/internal/structs/user_group.go

```diff
@@ -0,0 +1,12 @@
+package structs
+
+type SpaceAccessRule struct {
+	Space            string `graphql:"space"`
+	SpaceAccessLevel string `graphql:"spaceAccessLevel"`
+}
+
+type UserGroup struct {
```

Comment:
The user group should have a status (inactive or active).

Reply:
Same as above.

Reply:
That's fine with me.